config/deploy.rb

```diff
@@ -80,6 +80,37 @@ def set_branch
 before "deploy:assets:precompile", "deploy:yarn_install"
 before "deploy:assets:precompile", "deploy:whenever"
 
+namespace :application do
+  # You can/ should apply this command to a single host
+  # cap --hosts=figgy-web-staging1.princeton.edu staging application:remove_from_nginx
+  desc "Marks the server(s) to be removed from the loadbalancer"
+  task :remove_from_nginx do
+    count = 0
+    on roles(:app) do
+      count += 1
+    end
+    if count > 1
+      raise "You must run this command on individual servers utilizing the --hosts= switch"
```

Comment:
Could we allow taking down half the machines at once? especially with figgy, there are a lot of boxes.

Reply:
Good idea, implemented.